backend/functions/Alessio-Revenue/data/repository.js

```diff
@@ -124,18 +140,20 @@ export class Repository {
       .where("booking.hostId = :hostId", { hostId: cognitoUserId });
 
     if (startDate && endDate) {
-      query = query.andWhere(
-        `to_timestamp(booking."arrivaldate" / 1000) BETWEEN :startDate AND :endDate`,
-        { startDate, endDate }
-      );
+      query = query.andWhere(`to_timestamp(booking."arrivaldate" / 1000) BETWEEN :startDate AND :endDate`, {
+        startDate,
+        endDate,
+      });
     }
 
     const result = await query
-      .select(`
+      .select(
+        `
         AVG(
           DATE_PART('day', to_timestamp(booking."departuredate" / 1000) - to_timestamp(booking."arrivaldate" / 1000))
         ) AS "averageLengthOfStay"
-      `)
+      `
+      )
```

Comment:
```suggestion
      .select(`
       AVG(
         DATE_PART('day', to_timestamp(booking."departuredate" / 1000) - to_timestamp(booking."arrivaldate" / 1000))
       ) AS "averageLengthOfStay"
      `)
```
zelfde comment als hierboven, btw dit is meer een mening comment, als je deze niet zou willen toepassen vind ik dat niet erg.